src/matchers/toStartWith/predicate.js

```diff
@@ -0,0 +1 @@
+export default (prefix, string) => prefix === string.substring(0, prefix.length);
```

Comment:
You can also use `string.startsWith(prefix)` here. Your solution is fine too :smile:

Reply:
Wow, that's really nice. I switched it over.